interfaces/InputDeviceCapabilities.idl

```diff
@@ -0,0 +1,22 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the
+// "Input Device Capabilities" spec.
+// See: https://wicg.github.io/InputDeviceCapabilities/
+
+[Constructor(optional InputDeviceCapabilitiesInit deviceInitDict)]
+interface InputDeviceCapabilities {
+    readonly attribute boolean firesTouchEvents;
+    readonly attribute boolean pointerMovementScrolls;
+};
+
+dictionary InputDeviceCapabilitiesInit {
+    boolean firesTouchEvents = false;
+    boolean pointerMovementScrolls = false;
```

Comment:
Even Chrome hasn't implemented this bit :). But I guess from the test point of view this would be alright.

Reply:
Yep, I noticed it was failing. But correctly so, should be removed from spec if it stays failing in all browsers forever, so having the test can help surface that later :)